### PR TITLE
fixing issue with recent Daikin firmware update

### PR DESCRIPTION
**Pull request by cbrandlehner**
The Daikin firmware is now requiring a User-Agent string. There seems to be a bug in the firmware, requiring that the 'Host' request is proper case, not 'HOST' or 'host'.
fixing: #21

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -112,7 +114,10 @@ Daikin.prototype = {
 	getCurrentHeatingCoolingState: function(callback) {
 		this.log("getCurrentHeatingCoolingState from:", this.apiroute+"/aircon/get_control_info");
 		request.get({
-			url: this.apiroute+"/aircon/get_control_info"
+			url: this.apiroute+"/aircon/get_control_info",
+      headers: {
+                 'User-Agent' : 'request', 'Host' : this.apiIP
```
Comment (kennyadsl) [inline index.js:119]: What about `homebridge` or `homebridge-daikin` as `User-Agent`?

File: index.js [hunk 2]
```diff
@@ -35,8 +35,10 @@ function Daikin(log, config) {
 
 	this.name = config.name;
 	this.apiroute = config.apiroute || "apiroute";
-	this.log(this.name, this.apiroute);
-	
+        this.apiIP = this.apiroute.replace(/^https?:\/\//,'');
```
Comment (kennyadsl) [inline index.js:38]: Can we use https://nodejs.org/api/url.html#url_url_hostname?

**Review by kennyadsl — COMMENTED** [review]
Thanks! I was looking at https://github.com/request/request/pull/1211 and I think PR this will fix the issue!

I left some comments, not sure about the `url.hostname` one since I'm new in the Node.js world. Also indentation is a bit off, maybe it's some editor issue?

File: index.js (hunk 2 above)
Comment (crunchy59) [reply index.js:38]: Hello did you have find the solution? Because since the last firmware daikin not working with homebridge... Thanks.

Comment (kennyadsl) [reply index.js:38]: Yes please use this one: https://github.com/cbrandlehner/homebridge-daikin-local

It’s working for me.

Comment (crunchy59) [reply index.js:38]: Many thanks I will test.

**Pull request closed without merging**